mlos_bench/mlos_bench/config/services/remote/azure/scripts/setup-rg/setup-rg.ps1

```diff
@@ -0,0 +1,118 @@
+

```

Comment:
I think you might need to run `make licenseheaders` to apply the standard license header to this file before it will pass the CI checks.

Reply:
The `licenseheaders` tool does not seem to recognize `.ps1` files despite us including it as an extension, but I manually added the license header in

Reply:
You are correct.  I guess we'd just been doing that manually.

I've made a PR to fix that upstream:
https://github.com/johann-petrak/licenseheaders/pull/78